torch/_dynamo/symbolic_convert.py

```diff
@@ -506,6 +517,82 @@ def inline_user_function_return(self, fn, args, kwargs):
             self.restore_graphstate(state)
             raise
 
+    # `create_call_function_at` requires the following:
+    def create_call_function_at(
```

Comment:
Do we need to cache this similarly to `create_call_resume_at`? Perhaps if the function is called in a loop?